scripts/bfasst/compare_waveforms/file_parsing/parse_diff.py

```diff
@@ -21,39 +26,46 @@ def parse_io(line, input_output, words, new_word):
     branches in parse_signals"""
 
     word = ""
+
     for i in line:
+
         if i == " ":
             new_word = True
+
         if new_word is False:
             word = word + i
+
         else:
+
             if (word != "$var") & (word != input_output):
```

Comment:
I'm pretty sure this ampersand is a bitwise operator in python, you probably want `and`.